utils/schemas/main_config.json

```diff
@@ -370,6 +375,77 @@
       "examples": [
         "https://www.newrelic.com"
       ]
+    },
+    "documentation": {
+      "$id": "#/properties/documentation",
+      "type:": "array",
```

Comment:
extra colon in the field name.

Reply:
🤦  thanks! i'm now going to make an eye appointment ;)